capstone-rs/src/instruction.rs

```diff
@@ -4,13 +4,16 @@ use core::marker::PhantomData;
 use core::ops::Deref;
 use core::slice;
 use core::str;
+use alloc::vec::Vec;
 
 use capstone_sys::*;
 
 use crate::arch::ArchDetail;
 use crate::constants::Arch;
 use crate::ffi::str_from_cstr_ptr;
 
+pub type CsRegs = [u16; 64];
```

Comment:
Remove this since there's no need to redefine the `cs_regs` type. This type does not need to be public.